web/sites/default/config/filter.format.simple_editor.yml

```diff
@@ -16,7 +16,7 @@ filters:
     status: true
     weight: -10
     settings:
-      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol type start> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <p> <hr> <sup> <sub> <s> <a href hreflang !href accesskey id rel target title> <i class=""> <br> <drupal-entity data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align data-caption data-embed-button data-langcode alt title> <table class> <caption> <tbody> <thead class> <tfoot class> <th class> <td class> <tr class> <abbr title> <caption class>'
+      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <p lang=""> <div lang=""> <span lang=""> <a lang=""> <ol type start> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <p> <hr> <sup> <sub> <s> <a href hreflang !href accesskey id rel target title> <i class=""> <br> <drupal-entity data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align data-caption data-embed-button data-langcode alt title> <table class> <caption> <tbody> <thead class> <tfoot class> <th class> <td class> <tr class> <abbr title> <caption class>'
```

Comment:
The `a` attribute was duplicated in your change. Also, looks like the only change we need to make is to add `span` as lang is allowed without explicitly declaring it.

Reply:
Oh... you duplicated the `p` element as well. I didn't see if `div` was duplicated. 🤔

Reply:
Ah ok, I didn't realize that. Thanks for the flag, will fix.